ios/RNSScreenStackHeaderConfig.mm

```diff
@@ -1,7 +1,15 @@
-#import "RNSScreenStackHeaderConfig.h"
-#import "RNSScreen.h"
-#import "RNSSearchBar.h"
-
+#ifdef RN_FABRIC_ENABLED
+#import <React/RCTConversions.h>
+#import <React/RCTFont.h>
```

Comment:
This import is in both archs

Reply:
[6818798](https://github.com/software-mansion/react-native-screens/pull/1417/commits/68187987c0a9dd767ad4df46d25bd205fc0781f1)